API/api/models.py

```
@@ -1,3 +1,93 @@
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    class Meta:
+        db_table = "events"
+
+    def __str__(self) -> str:
+        return f"{self.name} ({self.event_date})"
+
+
+class Plage(models.Model):
+    event = models.ForeignKey(Event, on_delete=models.CASCADE, related_name="plages")
+    name = models.CharField(max_length=120, blank=True, null=True)  # optionnel
```

Ruff (0.14.14)

[warning] Avoid using `null=True` on string-based fields such as `CharField`

(DJ001)